src/Estimators/src/LeggedOdometry.cpp

```diff
@@ -905,3 +905,26 @@ bool LeggedOdometry::Impl::changeFixedFrame(const iDynTree::FrameIndex& newIdx,
 }
 
 
+bool LeggedOdometry::changeFixedFrame(const std::ptrdiff_t& newIdx,
+                                      const Eigen::Quaterniond& frameOrientationInWorld,
+                                      const Eigen::Vector3d& framePositionInWorld)
```

Comment:
Why not eigen::ref?

Reply:
I thought about it, but here I followed an existing signature in the FloatingBaseEstimators.

Reply:
ok